src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type BackendCall <: backend.BackendCall
```

Comment:
There's a pretty strong anti-ALL_CAPS sentiment in scala, which I completely agree with. You can always say that this is covered by the "simple" case in the official style guide and thus I'll go back to `A` :)